AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -200,6 +200,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/72902
+uploaded.net#%#//scriptlet('adjust-setTimeout')
```

Comment:
@wout-codes 
please add more arguments to this scriptlet to make the rule more specific, for example try:
`uploaded.net#%#//scriptlet("adjust-setTimeout", "", "1000", "0.02")`